frontend/src/pages/Form/Form.styled.ts

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+import Button from 'components/atoms/Button';
+import { colors } from 'styles/colors';
+
+export const StyledContainer = styled.div`
+  width: 100%;
+  height: 100%;
+`;
+
+export const StyledTopWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 80px;
+  width: 100%;
+  padding: 10px;
+  border-bottom: 1.5px solid lightgray;
+`;
+
+export const StyledBottomWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+`;
+
+export const StyledForm = styled.form`
+  padding: 100px;
+  width: 100%;
+  margin: 0;
+  @media (min-width: 768px) {
```

Comment:
In `geeks` we had something related to media queries. Could you check how it was working there?